### ISSUE TITLE
Lack of tinkers installed causes console spam of recipe unable to be parsed for tinkers compat recipes

### ISSUE THREAD

**Issue by DelviousCrafts** [MrScautHD/Beyond-Earth#236]
### Mod loader

Forge

### Minecraft version

1.19

### Beyond Earth version

6.4c

### Modloader version

Forge: 43.2.8

### The latest.log file

https://gist.github.com/DelviousCrafts/79168c707259a6d7b0bfc30120083bc1

### Issue description

Conditions in the recipe for compat with Tinkers cause spam in the console of inability to parse the recipe when Tinkers is not installed. This is due to a condition that is only present in Tinkers, and if Tinkers is not installed the recipe parser does not recognize the condition. A way to fix this is to use a conditionally loaded data recipe, instead of simply a double condition, checking to see if Tinkers is installed first, then using the condition within the recipe if the condition succeeds. An example of this fix is attached. 

[ore_singular.zip](https://github.com/MrScautHD/Beyond-Earth/files/11171819/ore_singular.zip)

### Steps to reproduce

1. Install Beyond Earth without Tinkers
2. Load game and find spam of recipe errors in console

**Comment by dikivan2000** [comment]
@DelviousCrafts does this interfere with gameplay without tinkers by any chance? Can I just ignore these errors?

**Comment by DelviousCrafts** [comment]
Yes, it can be ignored as it causes no gameplay issues. However, it can also be negated completely by using the format in the example [here](https://gist.github.com/DelviousCrafts/2c6e9a0eb998c35334f891a01f447d1c) by calling the condition to check if the mod is loaded before bothering to load the recipe.

**Comment by dikivan2000** [comment]
Thanks for the info. There seems to be more issues like this (#256) that the dev calls "normal" in Discord so I ended up avoiding the mod altogether. Maybe next time when it's a bit more polished...

**Comment by st0x0ef** [comment]
I never said that it's normal...

**Comment by dikivan2000** [comment]
> I never said that it's normal...

I beg to differ. https://discord.com/channels/698598471896268931/1006528459801100478/1139460727594287144

**Comment by st0x0ef** [comment]
OK yes

What I want to say is that I know this issue and it's normal that you're get this error. It's not my priority for now to fix that since I've a lot more of things to do now
